cirq-core/cirq/work/observable_measurement_test.py

```diff
@@ -509,37 +509,29 @@ def test_measure_grouped_settings_read_checkpoint(tmpdir):
             grouped_settings=grouped_settings,
             sampler=cirq.Simulator(),
             stopping_criteria=cw.RepetitionsStoppingCriteria(1_000, repetitions_per_chunk=500),
-            checkpoint=True,
-            checkpoint_fn=f'{tmpdir}/obs.json',
-            checkpoint_other_fn=f'{tmpdir}/obs.json',  # Same filename
+            checkpoint=CheckpointFileOptions(
+                checkpoint=True,
+                checkpoint_fn=f'{tmpdir}/obs.json',
+                checkpoint_other_fn=f'{tmpdir}/obs.json',  # Same filename
+            ),
         )
     _ = cw.measure_grouped_settings(
         circuit=circuit,
         grouped_settings=grouped_settings,
         sampler=cirq.Simulator(),
         stopping_criteria=cw.RepetitionsStoppingCriteria(1_000, repetitions_per_chunk=500),
-        checkpoint=True,
-        checkpoint_fn=f'{tmpdir}/obs.json',
-        checkpoint_other_fn=f'{tmpdir}/obs.prev.json',
+        checkpoint=CheckpointFileOptions(
+            checkpoint=True,
+            checkpoint_fn=f'{tmpdir}/obs.json',
+            checkpoint_other_fn=f'{tmpdir}/obs.prev.json',
+        ),
     )
     results = cirq.read_json(f'{tmpdir}/obs.json')
     (result,) = results  # one group
     assert result.n_repetitions == 1_000
     assert result.means() == [1.0]
 
 
-def _test_parse_stopping_criteria():
```

Comment:
Is this coverage now provided by another test?

Reply:
we removed this behavior from the previous round of review, so there's nothing to test